src/Google.Pubsub.V1/PublisherClient.cs

```diff
@@ -618,10 +619,14 @@ public virtual Publisher.IPublisherClient GrpcClient
         /// Lists matching topics.
         /// </summary>
         /// <param name="project">The name of the cloud project that topics belong to.</param>
+        /// <param name="pageToken">The token returned from the previous request. An empty string retrieves the first page.</param>
+        /// <param name="pageSize">The size of page to request, the response will not be larger than this, but may be smaller. 0 uses a server-defined page size.</param>
```

Comment:
Done. pageSize is now nullable. Null or 0 now use a server-defined page size.
Also pageToken now has a default of null. Null or empty string return the first page.